.gitignore

```diff
@@ -7,4 +7,4 @@
 
 .refena_inspector/
 
-/secrets
+/secrets
```

Comment:
If think you removed the line break here.

Reply:
> If think you removed the line break here.

Oh, I was going to ignore the flutter folder, but I decided to delete it so I removed it.